### PR TITLE
Bug fix in signup and profile and posting image functionality

### PR DESCRIPTION
**Pull request by Developers-codz**
Posting image feature, and profile picture update is added, Bug fixing in signup and profile also done.
Please check the functioning and if found any bug or UI is breaking do let me know

Thank you for reviewing my PR :)

### CODE REVIEW

**Review by vishalg8454 — CHANGES_REQUESTED** [review]
2 bugs i noticed

- Unable to edit own posts from profile page
- No option for text only posts, image is necessary. This is not good experience.

**Comment by Developers-codz** [comment]
> 2 bugs i noticed
> 
> * Unable to edit own posts from profile page
> * No option for text only posts, image is necessary. This is not good experience.

Hey, thanks for noticing those, I have fixed the first bug and for the second one its not a bug I took reference from Instagram and there we can't post without image that's why

File: src/Redux/Reducers/authSlice.js [hunk 1]
```diff
@@ -32,6 +33,29 @@ export const signup = createAsyncThunk("/auth/signup", async (userDetail) => {
...
+          { userData },
+          {
+            headers: {
+              authorization: encodedToken,
+            },
+          }
+        );
+      SuccessToast("User Data Updated Successfully")
+      console.log(response.data)
+        return response.data;
```
Comment (vishal804) [inline src/Redux/Reducers/authSlice.js]: Console.log can be removed

**Review by vishal804 — APPROVED** [review]
Code Looks Clean

**Pull request merged**